x-pack/plugins/security_solution/public/resolver/view/use_camera.ts

```diff
@@ -284,14 +284,30 @@ export function useAutoUpdatingClientRect(): [DOMRect | null, (node: Element | n
   const [rect, setRect] = useState<DOMRect | null>(null);
   // Using state as ref.current update does not trigger effect hook when reset
   const [currentNode, setCurrentNode] = useState<Element | null>(null);
-
+  const [currentY, setCurrentY] = useState<number>(0);
+  const [currentX, setCurrentX] = useState<number>(0);
   const ref = useCallback((node: Element | null) => {
     setCurrentNode(node);
     if (node !== null) {
       setRect(node.getBoundingClientRect());
     }
   }, []);
   const { ResizeObserver } = useContext(SideEffectContext);
+  const handleScroll = useCallback(() => {
+    if (currentX !== window.scrollX || currentY !== window.scrollY) {
+      if (currentNode !== null) {
+        setRect(currentNode.getBoundingClientRect());
+        setCurrentY(window.scrollY);
+        setCurrentX(window.scrollX);
+      }
+    }
+  }, [currentX, currentY, setCurrentX, setCurrentY, currentNode]);
+  useEffect(() => {
+    window.addEventListener('scroll', handleScroll, { passive: true });
+    return () => {
+      window.removeEventListener('scroll', handleScroll);
```

Comment:
:red_circle: I _think_ you might also need to supply `{ passive: true }` here for it to treat it as a "matching" signature and do the removal? I'll test and get back to you in a sec on that.

Reply:
no, passive is only a possible value for the options object in addEventListener, not removeEventListener

Reply:
OK, nevermind looks like that's just for `useCapture` / `{capture: true}`

Reply:
The advice for this on MDN reads as:

>It's worth noting that some browser releases have been inconsistent on this, and unless you have specific reasons otherwise, it's probably wise to use the same values used for the call to addEventListener() when calling removeEventListener().

https://wiki.developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener

Reply:
You can leave this one be. I'm not sure how relevant that advice on MDN is anymore - probably "not very".